test/runner/lib/classification.py

```diff
@@ -348,13 +348,19 @@ def _classify(self, path):
                 'network-integration': target.name if 'network/' in target.aliases else None,
             }
 
-        if path.startswith('test/integration/'):
+        if path.startswith('test/integration/network-'):
```

Comment:
Handling of the `test/integration/` directory should not be removed. It's there to fall back on when there isn't a more specific match. Also, there's not much point in adding special handling for `network-` prefixed files, since there's only a single matching file which is very infrequently updated, so I'd just eliminate this change.